src/components/RankSection/RankSection.js

```diff
@@ -0,0 +1,28 @@
+import { getScores } from '../../model/saveScore';
+import RankRecord from '../RankRecord/RankRecord';
+
+const RankSection = (categoryName) => {
+  const rankSection = document.createElement('section');
+  rankSection.classList.add('rankSection');
+
+  const category = document.createElement('h2');
+  category.classList.add('rankSection__category');
+  category.textContent = `Ranking: ${categoryName}`;
+
+  const ranks = document.createElement('div');
+  ranks.classList.add('rankSection__ranks');
+
+  const scores = getScores(categoryName);
+  const rankRecords = scores
+    .sort((thisPlayer, nextPlayer) => (thisPlayer.score > nextPlayer.score ? -1 : 1))
+    .slice(0, scores.length > 20 ? 20 : scores.length)
+    .map((playerScore, index) => RankRecord(index + 1, playerScore));
+
+  ranks.append(...rankRecords);
```

Comment:
czy dobrze rozumiem - ta zmienna przechowuje scores posortowane, obcięte do długości max 20 i mapuje wynik z pozycją w rankingu?
a linia 21? chodzi o zapis ...rankRecords :)

Reply:
Wygląda to tak linia po linii:
  16.  Weź scores
  17. Posortuj od największego do najmniejszego
  18. Wybierz max 20 wyników
  19. Zamienić to w elementy RankRecords.

Więc rankRecords przechowuje tablicę elementy wygenerowanych na podstawie 20 najwyższych wyników :)